### PR TITLE
add(temporary column sample table)

### PR DESCRIPTION
**Pull request by ChrOertlin**
## Description

This PR adds a column "calculated_read_count" to the Sample table in statusDB.

The reason for this is the parallel validation of read count caluclation using CGstats and the new SampleLaneSequencingMetrics.

Once validated, this column will be dropped and the calucaltion logic will be implemented into the reads column.

### Added

-

### Changed

-

### Fixed

-

### How to prepare for test

- [ ] Ssh to relevant server (depending on type of change)
- [ ] Use stage: `us`
- [ ] Paxa the environment: `paxa`
- [ ] Install on stage (example for Hasta):
    ```shell
    bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_main -t cg -b [THIS-BRANCH-NAME] -a
    ```

### How to test

- [ ] Do ...

### Expected test outcome

- [ ] Check that ...
- [ ] Take a screenshot and attach or copy/paste the output.

## Review

- [ ] Tests executed by
- [ ] "Merge and deploy" approved by
Thanks for filling in who performed the code review and the test!

### This [version](https://semver.org/) is a

- [ ] **MAJOR** - when you make incompatible API changes
- [ ] **MINOR** - when you add functionality in a backwards compatible manner
- [ ] **PATCH** - when you make backwards compatible bug fixes or documentation/instructions

## Implementation Plan

- [ ] Document in ...
- [ ] Deploy this branch on ...
- [ ] Inform to ...

### CODE REVIEW

File: cg/meta/demultiplex/demux_post_processing.py [hunk 1]
```diff
@@ -197,7 +197,7 @@ def update_single_sample_read_count(self, sample_id: str) -> None:
                 sample_internal_id=sample_id
             )
             LOG.info(f"Updating sample {sample_id} with read count {sample_read_count}")
-            sample.reads = sample_read_count
+            sample.calculated_read_count = sample_read_count
```
Comment (islean) [inline cg/meta/demultiplex/demux_post_processing.py:200]: So what happens to the reads column in this case?

**Review by islean — APPROVED** [review]
Looks nice! Just one question - is this intended to be deployed to production? Will it cause any differences in functionality that the temporary column is filled instead of reads?

File: cg/meta/demultiplex/demux_post_processing.py (hunk 1 above)
Comment (ChrOertlin) [reply cg/meta/demultiplex/demux_post_processing.py:200]: Nothing, it is updated with the CGstats logic that we aim to deprecate. After some reviewing of the code there are some more connections that we need to sever one by one to have this all go away. So a parallel approach in this case might be healthier.

**Review by seallard — APPROVED** [review]
Nice 👍  This will make it possible to run the new finish command in parallel with the old one until we have removed all dependencies on cgstats and can can replace the old command completely.

**Comment by ChrOertlin** [comment]
Tested in stage:
image

**Pull request merged**